src/main/resources/com/puppycrawl/tools/checkstyle/messages.properties

```diff
@@ -1,5 +1,6 @@
 Checker.finishLocalSetup=if no custom moduleFactory is set, moduleClassLoader must be specified
-Checker.processFilesException=Exception was thrown while processing {0}
+Checker.processFilesException=Exception was thrown while processing {0}. Set ''skipFileOnJavaParseException''\
+    property of ''TreeWalker'' module to ''true'' to skip all these exceptions.
```

Comment:
If this message works, I will translate this to other languages as well.

Reply:
We should not change this message, we should not promote this property to be used.
User need to understand context of problem to make decision himself, as this skip can result in more uncertainty and confusion.

Reply:
removed